packages/gestalt/src/Datapoint.js

```diff
@@ -0,0 +1,77 @@
+// @flow strict
+import React, { type Node } from 'react';
+import PropTypes from 'prop-types';
+import Text from './Text.js';
+import Heading from './Heading.js';
+import Flex from './Flex.js';
+import IconButton from './IconButton.js';
+import Tooltip from './Tooltip.js';
+import DatapointTrend from './DatapointTrend.js';
+
+type TrendObject = {|
+  accessibilityLabel: string,
+  value: number,
+|};
+
+type Props = {|
+  title: string,
+  value: string,
+  trend?: TrendObject,
+  trendSignal?: 'good' | 'bad' | 'neutral' | 'auto',
+  infoText?: string,
+  size?: 'md' | 'lg',
+|};
+
+export default function Datapoint({
+  title,
+  value,
+  trend,
+  infoText,
+  trendSignal = 'auto',
+  size = 'md',
+}: Props): Node {
+  const infoTextNode = infoText ? (
+    <Tooltip text={infoText}>
+      <IconButton accessibilityLabel="" size="sm" icon="info-circle" iconColor="gray" padding={1} />
+    </Tooltip>
+  ) : null;
+
+  const valueSize = size === 'lg' ? 'lg' : 'sm';
+  const percentChangeGap = size === 'lg' ? 4 : 2;
+
+  return (
+    <Flex gap={1} direction="column">
+      <Flex gap={1} alignItems="center" minHeight={24}>
+        <Text size="sm">{title}</Text>
+        {infoTextNode}
+      </Flex>
+      <Flex gap={percentChangeGap} alignItems="center">
+        <Heading accessibilityLevel="none" size={valueSize}>
+          {value}
+        </Heading>
+        {trend !== undefined && trend !== null ? (
```

Comment:
Given how `trend` is typed, we can just make a truthy check here:
```js
{trend && <DatapointTrend />}
```

Reply:
Done!